Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs

```diff
@@ -235,6 +247,9 @@ private void OnDestroy()
         // Set asset would usually only be called when the window is open
         private void SetAsset(InputActionAsset asset)
         {
+            if (asset == null)
+                return;
```

Comment:
This case looks malformed. .inputactions editor windows with no associated assets should get nuked, it seems. What's the flow where we run into this if?

Reply:
The OpenEditor method, which calls SetAsset, is public and takes the asset as an argument, so it seems better to check it for null. I didn't see that the editor window gets blown away if there's no asset. I was definitely able to open one with this being null.